salt/haproxy/errors/503-velum_database_not_ready.http

```diff
@@ -0,0 +1,73 @@
+HTTP/1.0 503 Service Unavailable
+Cache-Control: no-cache
+Connection: close
+Content-Type: text/html
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>SUSE CaaS Platform is initializing</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <style>
+    body {
+      background-color: #003258;
+      color: #2E2F30;
+      text-align: center;
+      font-family: Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif;
+      margin: 0;
+    }
+
+    div.dialog {
+      width: 95%;
+      max-width: 33em;
+      margin: 4em auto 0;
+    }
+
+    div.dialog > div {
+      border-bottom: 1px solid #BBB;
+      border-top: #00c081 solid 4px;
+      border-top-left-radius: 9px;
+      border-top-right-radius: 9px;
+      background-color: #00243E;
+      box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+    }
+
+    div.dialog > div > p {
+      color: #fff;
+    }
+
+    h1 {
+      font-size: 100%;
+      color: #fff;
+      line-height: 1.5em;
+      font-weight: 500;
+      font-size: 18px;
+    }
+
+    div.dialog > p {
+      margin: 0 0 1em;
+      padding: 1em;
+      background-color: #F7F7F7;
+      border: 1px solid #CCC;
+      border-right-color: #999;
+      border-left-color: #999;
+      border-bottom-color: #999;
+      border-bottom-left-radius: 4px;
+      border-bottom-right-radius: 4px;
+      border-top-color: #DADADA;
+      color: #333333;
+      box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+    }
+  </style>
+</head>
+
+<body>
+  <div class="dialog">
+    <div>
+      <h1>SUSE CaaS Platform is initializing</h1>
```

Comment:
If the rule on haproxy is to show this warning in the case of a 503 error we cannot be sure that it's always initializing. It could happen that the backend is down for some reason, and in that case we don't want to show that the platform is initializing. This message should be much more generic, something like "SUSE CaaS Platform is not ready, please wait" or something along those lines.